pkg/tag/generate_tag_definitions.py

```diff
@@ -73,3970 +53,14 @@ def generate(out: IO[str]):
     print("	if len(tagDict) > 0 {", file=out)
     print("		return", file=out)
     print("	}", file=out)
-    print("	tagDict = make(map[Tag]TagInfo)", file=out)
+    print("	tagDict = make(map[Tag]Info)", file=out)
```

Comment:
non-blocking nit: might be worth writing out this block as a multiline python string, but up to whatever you consider the most readable!

Reply:
Done.